cellxgene_schema_cli/cellxgene_schema/validate.py

```diff
@@ -1076,47 +1033,20 @@ def _validate_raw(self):
         # If all checks passed then proceed with validation
         if all(checks):
 
-            normalization = self.adata.uns["X_normalization"]
-
             # If both "raw.X" and "X" exist but neither are raw
             if not self._is_raw() and self._get_raw_x_loc() == "raw.X":
                 self.warnings.append(
                     "Raw data may be missing: data in 'raw.X' contains non-integer values."
                 )
 
-            # If raw data is missing: no "raw.X", and "X_normalization" is not "none"
-            if (
-                not self._is_raw()
-                and self._get_raw_x_loc() == "X"
-                and normalization != "none"
-            ):
-
-                self.errors.append(
-                    "Raw data is missing: there is no 'raw.X' and 'X_normalization' is not 'none'."
-                )
-
-            # If raw data is in X but X_normalization is NOT none raise an error
-            if (
-                self._is_raw()
-                and self._get_raw_x_loc() == "X"
-                and normalization != "none"
-            ):
-
-                self.errors.append(
-                    f"uns['X_normalization'] is '{normalization}' but raw data seems to be in X, "
-                    f"if X is raw then uns['X_normalization'] MUST be 'none'."
-                )
-
             # If raw data is in X and there is nothing in raw.X (i.e. normalized values are not provided), then
             # add a warning because normalized data for RNA data is STRONGLY RECOMMENDED
             if (
                 self._is_raw()
                 and self._get_raw_x_loc() == "X"
-                and normalization == "none"
             ):
-
                 self.warnings.append(
```

Comment:
Should this be an error? @brianraymor 

```suggestion
                self.error.append(
```